src/pages/about.js

```diff
@@ -217,27 +218,39 @@ const AboutUs = ({ data: { firstSectionImage, secondSectionImage } }) => {
             </div>
           </section>
 
-
         </div>
-        <Team />
+        <Team
+          billyPicture={billyPicture}
+          danielPicture={danielPicture}
+          eliePicture={eliePicture}
+          willyPicture={willyPicture}
+          alainPicture={alainPicture}
```

Comment:
@willyserge , what if the team grows? Will you always create new props?

Reply:
@eliemugenzi , this seemed convenient since the gatsby image api only works on page components. any suggestions ?

Reply:
How about you add a `members` prop as an array of all of us instead of creating a prop for each and everyone?